pypxe/tftp.py

```diff
@@ -123,13 +129,16 @@ def read(self, address, message):
             self.logger.error("Mode '{mode}' not supported".format(mode = mode))
             self.tftpError(address, 5, 'Mode {mode} not supported'.format(mode = mode))
             return
-        filename = self.filename(message)
+        req_file = self.filename(message)
+        # avoid directory traversal: strip all ../ and make it relative
+        filename = os.path.normpath(os.sep + os.getcwd() + os.sep + req_file).lstrip(os.sep)
```

Comment:
This doesn't works on windows due the presence of c: in the top of `os.getcwd()`.
I don't see many simple and portable ways to protect against directory traversal without chroot().
My proposal is to add a warning about the issue when catching the os.chroot() exception.

What do you think ?